mlflow/tracking/fluent.py

```diff
@@ -257,7 +258,7 @@ def set_tags(tags):
 
 def log_artifact(local_path, artifact_path=None):
```

Comment:
We should add tests for the behavior of this method when it's invoked on a directory.

Reply:
Would it suffice to change existing tests on `log_artifacts` to use `log_artifact`?